api/src/routes.ts

```diff
@@ -251,17 +231,11 @@ app.get('/send-token/', (req, res) => {
 
 app.get('/get-token/', async (req, res) => {
   if (req.user) {
-    const signingKey = await KEY_SERVICE.getSigningKey();
-    if (signingKey === null || signingKey === undefined) {
+    try {
+      const token = await generateUserToken(req.user);
+      res.send(token);
+    } catch {
       res.status(500).send('Signing key not set up');
```

Comment:
does this error still make sense - could be other reasons generateUserToken fails?